.buildkite/scripts/common.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+set -euo pipefail
+
+WORKSPACE="$(pwd)/bin"
+
+create_workspace() {
+    if [[ ! -d "${WORKSPACE}" ]]; then
+    mkdir -p ${WORKSPACE}
+    fi
+}
+
+with_go() {
+    echo "Setting up the Go environment..."
+    retry 5 curl -sL -o ${WORKSPACE}/gvm "https://github.com/andrewkroh/gvm/releases/download/${SETUP_GVM_VERSION}/gvm-linux-amd64"
```

Comment:
Missing the definition of this retry function.
Should this function be added to this script ?